Marlin/Marlin_main.cpp

```diff
@@ -8434,6 +8435,21 @@ inline void gcode_M211() {
   SERIAL_ECHOLNPAIR(" " MSG_Z, soft_endstop_max[Z_AXIS]);
 }
 
+/**
+ * M217: Send text to serial output
+ *
+ * Usage: M217 some text
+ * Text must be prefixed by "//" and this will be prepended if it is not present
+ */
+inline void gcode_M217() {
+  if (strncmp(parser.string_arg, "//", 2) == 0) {
+    SERIAL_ECHOLN(parser.string_arg);
+  } 
+  else {
+    SERIAL_ECHOLNPAIR("// ", parser.string_arg);
+  } 
+}
```

Comment:
This will result in smaller and faster code:
```cpp
if (parser.string_arg[0] != '/' || parser.string_arg[1] != '/')
  SERIAL_ECHOPGM("// ");
SERIAL_ECHOLN(parser.string_arg);
```